# joint angles
std_msgs/Float64 joint0
std_msgs/Float64 joint1
std_msgs/Float64 joint2
std_msgs/Float64 joint3
std_msgs/Float64 joint4
std_msgs/Float64 joint5
std_msgs/Float64 joint6
std_msgs/Float64 joint7
